crucible/src/Lang/Crucible/Backend/Assumptions.hs

```diff
@@ -48,6 +48,7 @@ import           Control.Lens (Traversal, folded)
 import           Data.Kind (Type)
 import           Data.Functor.Identity
 import           Data.Functor.Const
+import           Data.Parameterized.TraversableF
```

Comment:
No need to block on this change, but generally I'd rather see qualified / explicit imports for all the reasons listed here: https://stackoverflow.com/questions/2386714/why-is-import-bad
```suggestion
import qualified Data.Parameterized.TraversableF as TF
```